src/App.tsx

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+const filtrateMovies = (movies: Movie[], query: string) => {
+  return movies.filter(movie => (
+    movie.description.toLowerCase().includes(query.toLowerCase())
+    || movie.title.toLowerCase().includes(query.toLowerCase())
+  ));
+};
+
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = filtrateMovies(moviesFromServer, query);
```

Comment:
do you really have to pass moviesFrom Server? To me it looks like it is imported and function already has an access to it